lib/config-list.ts

```diff
@@ -78,6 +79,8 @@ export function updateConfigsList(
   configFormat: ConfigFormat,
   ignoreConfig: readonly string[],
 ): string {
+  const { endOfLine } = context;
```

Comment:
Any variables needed from the context are always put in a destructuring statement at the top line of a function, for consistency.